plugins/woocommerce/src/StoreApi/SessionHandler.php

```diff
@@ -114,12 +114,4 @@ public function save_data() {
 		}
 	}
 
-	/**
-	 * Return the current session data.
-	 *
-	 * @return array
-	 */
-	public function get_session_data() {
-		return $this->_data ?? [];
-	}
```

Comment:
This is added due to the error I mentioned in the PR description https://github.com/woocommerce/woocommerce/pull/61216#discussion_r2406559357

FYI @RadoslavGeorgiev and @mikejolley as both of you were in this discussion.